src/css/cfpb-chart-builder.less

```diff
@@ -57,7 +57,7 @@
 
   .chart-label {
     font-size: @base-font-size-px;
-    .u-webfont-regular();
+    font-family: @webfont-regular;
```

Comment:
This should be `font-family: inherit;` to revert to the setting of "Avenir Next" on `body`.

Reply:
Fixed! Thanks! How'd you figure that out?

Reply:
Inspected and saw that your original rule was resulting in `Arial, Arial, sans-serif;`, so the `@webfont-regular` var wasn't set correctly. Removing the rule entirely resulted in things using Highchart's default of Lucida, so the rule has to be there. I knew that cf-core was setting Avenir Next on `body`, so `inherit` restores it to that from the Highcharts style.